crons.go

```diff
@@ -91,19 +91,20 @@ func runSentryCronsCheckin(ctx context.Context, job *batchv1.Job, eventHandlerTy
 		return errors.New("cannot find cronJob data")
```

Comment:
Please update the comment to the `runSentryCronsCheckin` function (seems to be out-of-date now)